contracts/references/aa-4337/VerifyingPayMaster.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+/* solhint-disable reason-string */
+
+import "./BasePaymaster.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+
+/**
+ * A sample paymaster that uses external service to decide whether to pay for the UserOp.
+ * The paymaster trusts an external signer to sign the transaction.
+ * The calling user must pass the UserOp to that external signer first, which performs
+ * whatever off-chain verification before signing the UserOp.
+ * Note that this signature is NOT a replacement for wallet signature:
+ * - the paymaster signs to agree to PAY for GAS.
+ * - the wallet signs to prove identity and wallet ownership.
+ */
+contract VerifyingPayMaster is BasePaymaster, Initializable {
```

Comment:
rename this to VerifyingPaymaster 

make sure everywhere it goes paymaster without M caps

Reply:
done